core/activity/src/common.rs

```diff
@@ -39,10 +39,15 @@ pub struct QueryTimeoutCommandIndex {
 }
 
 #[derive(Deserialize, Debug)]
-pub struct QueryTimeoutMaxEvents {
+pub struct QueryEvents {
     /// number of milliseconds to wait
-    #[serde(rename = "timeout", default = "default_query_timeout")]
-    pub timeout: Option<f32>,
+    #[serde(rename = "appSessionId")]
```

Comment:
above comment is irrelevant
cc @mfranciszkiewicz @nieznanysprawiciel

Reply:
forthermore I'd use `#[serde(rename_all = "camelCase")]` which is not only more concise, but also less error-prone

Reply:
might be adressed in separate PR

Reply:
afair this code was auto-generated and only slightly modified later on